manage-data/lifecycle/index-lifecycle-management/policy-view-status.md

```diff
@@ -0,0 +1,132 @@
+1. Enable **Include hidden indices** to view all indices, including those managed by ILM. Note that if you're using data streams, you can find the data stream associated with any index listed in the **Data stream** column.
+1. Use the search tool to find the index you're looking for. You can also filter the results by lifecycle status and lifecycle phase.
+1. Select the index to view details.
+1. Open the **Index lifecycle** tab to view ILM details such as the current lifecycle phase, the ILM policy name, the current [index lifecycle action](elasticsearch://reference/elasticsearch/index-lifecycle-actions/index.md), and other details.
+
+   ![Index lifecycle status page](/manage-data/images/elasticsearch-reference-ilm-status.png "")
+
+To view the current lifecycle status for a datastream:
+
+1. Go to **Stack Management > Index Management** and open the **Data Streams** tab.
+1. Use the search tool to find the data stream you're looking for.
+1. Select the data stream to view details. In the flyout that opens, you have direct links to access the ILM policy and the index template.
+
+   ![Data stream status page](/manage-data/images/elasticsearch-reference-datastream-status.png "")
+:::
+
+:::{tab-item} API
+:sync: api
+Use the [Explain the lifecycle state API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-ilm-explain-lifecycle) to view the current lifecycle status for an index:
+
+```console
+GET my-index/_ilm/explain
+```
+
+The API response shows the current ILM phase and other details:
+
+```json
+{
+  "indices": {
+    ".ds-logs-elastic_agent.filebeat-default-2025.06.04-000001": {
+      "index": ".ds-logs-elastic_agent.filebeat-default-2025.06.04-000001",
+      "managed": true,
+      "policy": "logs",
+      "index_creation_date_millis": 1749060710541,
+      "time_since_index_creation": "19.05d",
+      "lifecycle_date_millis": 1749060710541,
+      "age": "19.05d",
+      "phase": "hot",
+      "phase_time_millis": 1749060711038,
+      "action": "rollover",
+      "action_time_millis": 1749060712038,
+      "step": "check-rollover-ready",
+      "step_time_millis": 1749060712038,
+      "phase_execution": {
+        "policy": "logs",
+        "phase_definition": {
+          "min_age": "0ms",
+          "actions": {
+            "rollover": {
+              "max_age": "30d",
+              "max_primary_shard_docs": 200000000,
+              "min_docs": 1,
+              "max_primary_shard_size": "50gb"
+            }
+          }
+        },
+        "version": 1,
+        "modified_date_in_millis": 1749059754275
+      }
+    }
+  }
+}
+```
+
+You can also call this API for a data stream:
+
+```console
+GET my-datastream/_ilm/explain
+```
+
+When called for a data stream, the API retrieves the current lifecycle status for the stream's backing indices:
+
+```json
+{
+  "indices": {
+    ".ds-logs-elastic_agent.filebeat-default-2025.06.04-000001": {
```

Comment:
`GET my-datastream` should return an index name `.ds-my-datastream...` not a copy of index-which-was-a-backing-datastream-did-you-intend-that? from just before

Reply:
Good point. I've updated the example to use real world indices and data streams from the System Integration (rather than 'my-index').